packages/docs/components/demos/DocsFlow.tsx

```diff
@@ -1,35 +1,31 @@
 import { DataStory } from '@data-story/ui'
-import { Application, core, coreNodeProvider, multiline, nodes, } from '@data-story/core';
+import { core, multiline, nodes, } from '@data-story/core';
 import { MockJSClient } from '../splash/MockJSClient';
 
 export default () => {
-  const app = new Application();
-
-  app.register(coreNodeProvider);
-
-  app.boot();
-
   const { Signal, Pass, Comment, Ignore } = nodes;
 
   const diagram = core.getDiagramBuilder()
-    .add({...Signal, label: 'DataSource'}, { period: 20, count: 100000})
-    .add({...Pass, label: 'Transforms'})
-    .add({...Ignore, label: 'Actions'})
-    .from('Pass.1.output').below('Ignore.1').add({...Pass, label: 'APIs'})
-    .add({...Ignore, label: 'Storage'})
-    .jiggle({x: 60, y: 25})
-    .above('Signal.1').add(Comment, { content: multiline`
+    .add({ ...Signal, label: 'DataSource' }, { period: 20, count: 100000 })
+    .add({ ...Pass, label: 'Transforms' })
+    .add({ ...Ignore, label: 'Actions' })
+    .from('Pass.1.output').below('Ignore.1').add({ ...Pass, label: 'APIs' })
+    .add({ ...Ignore, label: 'Storage' })
+    .jiggle({ x: 60, y: 25 })
+    .above('Signal.1').add(Comment, {
+      content: multiline`
       ### DataStory 🔥
       Combine data sources, transforms, actions, APIs, storage and more. Create custom nodes for your business logic.
-    `})
+    `
+    })
     .get()
 
   const client = new MockJSClient(diagram);
 
   return (
     <div className="w-full h-1/6">
       <DataStory
-        server={{ type: 'JS', app }}
+        server={{ type: 'JS' }}
```

Comment:
I thought this prop was removed. As "client" have all conversations with server and a client can be implemented in any way.. Is there still a need for server prop? If not, drop it. If there is a need, can it be passed into the Client via constructor as a dependency instead?

Reply:
The client is required property, so the server property can be removed. Should I go ahead and remove it in the next PR?

Reply:
Yes, le'ts!